internal/dag/dag.go

```diff
@@ -532,6 +532,22 @@ type SecureVirtualHost struct {
 	// only reason to set this to `true` is when you are migrating
 	// from internal to external authorization.
 	AuthorizationFailOpen bool
+
+	// BufferSettings specifies configuration for buffering request data sent to AuthorizationServer
+	AuthorizationServerBufferSettings *AuthorizationServerBufferSettings
```

Comment:
If we rename the API field per my previous comment, then a similar rename here would make sense for consistency.

Reply:
Done